code/modules/mob/say.dm

```diff
@@ -36,6 +38,14 @@
 
 	usr.emote("me",1,message,TRUE)
 
+/// This datum is used for listening to deadchat messages.
+/datum/deadchat_listener

```

Comment:
this should be a component

Reply:
I am turning the whole thing into a component at this time.